### ISSUE TITLE
ERROR: no root document

### ISSUE THREAD

**Issue by frizman21** [doorstop-dev/doorstop#442]
Following your example, I've successfully create a requirements project in a fresh git repo. 
```
mkdir fake_reqs
cd fake_reqs/
git init
Initialized empty Git repository in [working folder]/fake_reqs/.git/
fake_reqs $ doorstop add SRD
building tree...
ERROR: no document with prefix: SRD
fake_reqs $ doorstop create SRD ./reqs/srd
building tree...
created document: SRD (@/reqs/srd)
```

I'm attempting to add requirements to an existing repo, following the same steps and I'm running into an error: 
```
doorstop create SRD ./req/srd
building tree...
ERROR: no root document
```

Please advise...

**Comment by andreascian** [comment]
got the same issue here, using the latest version taken from PyPi (2.0.post2)

```
bash# pip install doorstop==2.0.post2
[...]
bash# doorstop create REQ ./reqs
building tree...
ERROR: no root document
```

This ones is also missing `six` dependency (which is not automatically installed by `pip`)

If I step back a couple of version, I found one that is working:

```
bash# pip install doorstop==1.5
[...]
bash# doorstop create REQ ./reqs
building tree...
created document: REQ (@/reqs)
```

It seems that 1.5 is the latest that is working correctly :thinking:

**Comment by fkromer** [comment]
Unfortunately I cannot use v1.5 do to dependency resolution issues. I could not find info about how the requirement yaml files should/may look like in the docs. In case someone else is looking for a minimalistic example of a valid yaml file the tests can be enlightening:

- doorstop's own requirement files: https://github.com/doorstop-dev/doorstop/tree/develop/reqs
- `.doorstop.yml`: https://github.com/doorstop-dev/doorstop/search?q=filename%3A.doorstop.yml&type=

**Comment by danieleades** [comment]
I'm getting the same error.

doorstop version v2.1.1

**Comment by jamesmunns** [comment]
I seem to have this problem when I have a virtualenv within the same folder as the requirements. I think maybe it's finding some requirements documents from the pip package?

If I make the virtualenv somewhere else on my filesystem, the setup commands work as expected

**Comment by jacebrowning** [comment]
I'll work on a fix to skip `.venv/` entirely.

**Comment by jacebrowning** [comment]
This should be fixed in v2.1.2.

**Comment by wienans** [comment]
@jacebrowning 
What about excluding all "." folders? 
We had the same issue as we used .env as the environment folder.
How likely is it that in .folder is something crutial for the application?
